.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file is obsolete I think. Editor specific

Reply:
Removed, sorry about that